src/routes/servers/server_create.rs

```diff
@@ -77,6 +77,8 @@ pub async fn req(user: User, info: Json<Data>) -> Result<Value> {
 
         icon: None,
         banner: None,
+
+        flags: None,
```

Comment:
what's the point of this addition?

Reply:
Wasn't this fixed in b4db321d0ef1ff296f6a611330cc1db73e5e566e already?

Reply:
Yeah it was, I cloned from an older commit.